tljh/config.py

```diff
@@ -183,11 +184,14 @@ def reload_component(component):
     from tljh import systemd, traefik
     if component == 'hub':
         systemd.restart_service('jupyterhub')
-        # FIXME: Verify hub is back up?
+        # Ensure hub is back up
+        while not systemd.check_service_active('jupyterhub'):
+            asyncio.sleep(1)
```

Comment:
Since this is a blocking function not a coroutine, let's use `time.sleep` here instead of `asyncio.sleep`